### PR TITLE
Show full name of the applicant in application list

### LINKED ISSUE #1351: Show applicant name, not eppn in the tables
Verify with LBR that this is ok, or make customizable

### PR DESCRIPTION
**Pull request by okahilak**
Closes #1351

### CODE REVIEW

**Comment by okahilak** [comment]
Never mind about this patch now, I noticed that there's some additional logic on back-end side that I need to take into account.

**Comment by okahilak** [comment]
This should now be ready for review.

File: src/cljs/rems/application.cljs [hunk 1]
```diff
@@ -461,8 +461,8 @@
        [:h3 (cond (= (:application/applicant application) user-id) (text :t.applicant-info/applicant)
                   (:userid attributes) (text :t.applicant-info/member)
                   :else (text :t.applicant-info/invited-member))]
-       (when-let [name (or (:commonName attributes) (:name attributes))]
-         [info-field (text :t.applicant-info/name) name {:inline? true}])
+;;       (when-let [applicant-name (get-applicant-name application)]
+;;         [info-field (text :t.applicant-info/name) applicant-name {:inline? true}])
```
Comment (hukka) [inline src/cljs/rems/application.cljs]: Commented left overs or accidental? get-applicant-name is not used elsewhere.

File: src/cljc/rems/application_util.cljc [hunk 2]
```diff
@@ -11,3 +13,8 @@
   (contains? (or (:possible-commands application) ;; TODO: remove v1 api usage
                  (:application/permissions application))
              :application.command/save-draft))
+
+(defn get-applicant-name [application]
+  (let [attributes (:application/applicant-attributes application)]
+    (when attributes
+      (first (filter identity (map attributes applicant-name-order))))))
```
Comment (hukka) [inline src/cljc/rems/application_util.cljc]: Maybe bring applicant-name-order closer, or inline (it's not too long) to make it easy to see what kind of data is expected to be in the applicant-attributes?

File: src/cljs/rems/application.cljs (hunk 1 above)
Comment (okahilak) [reply src/cljs/rems/application.cljs]: Done. The lines had been accidentally commented.

File: src/cljc/rems/application_util.cljc (hunk 2 above)
Comment (okahilak) [reply src/cljc/rems/application_util.cljc]: Done, it is now next to the function definition.

**Pull request merged**